script.js

```diff
@@ -180,6 +180,24 @@ workProjects.forEach((project, index) => {
 
```

Comment:
- Provide the requested styles guidelines in your Figma template.
**Requested style:**
<img width="547" alt="Screen Shot 2021-10-12 at 10 42 55" src="https://user-images.githubusercontent.com/22258274/136987889-aded1611-8716-4a79-8aad-c5da999c3586.png">

#

**Yours:**
<img width="402" alt="Screen Shot 2021-10-12 at 10 46 34" src="https://user-images.githubusercontent.com/22258274/136988183-b824aa7a-c001-49d3-98a5-80ff7310ae41.png">

**_ Kindly check your paddings_**